modules/webbrowser/src/webbrowsermodule.cpp

```diff
@@ -161,16 +161,18 @@ WebBrowserModule::WebBrowserModule(InviwoApplication* app)
         locale.erase(startErasePos, locale.find('.') - startErasePos);
     }
 
-    // Web helper executable should be located in Frameworks dir of bundle,
-    // see OS_MACOSX part in CMakeLists.txt
-    if (!filesystem::fileExists(subProcessExecutable)) {
-        subProcessExecutable =
-            cefParentDir +
-            std::string("/Frameworks/Inviwo Helper.app/Contents/MacOS/Inviwo CEF Helper");
-    }
 #else
     CefMainArgs args;
     CefSettings settings;
+    // Non-mac systems uses a single helper executable so here we can specify name
+    auto subProcessExecutable = exeDirectory + "/cef_web_helper." + exeExtension;
```

Comment:
this looks shady with the trailing `.` and most likely won't work on Linux (no exeExtension). Doesn't CMake take care of this?

Reply:
Probably right that we need to consider linux here as well. 
The CMake option would be to name the helper according to the main executable (inviwo/qt_minimal and so on) but I would rather not do that (although we are forced to do it on mac) because the module becomes application dependent.

Reply:
Any particular reason for doing not the same on other platforms?

Reply:
Should be fixed now

Reply:
Yes, they use three executables on Mac so it did not work with specifying a single path